app/views/feedbacks/index.html.haml

```diff
@@ -3,15 +3,15 @@
 
 = render 'shared/faq'
 
-= render '/shared/content_footer_ad'
```

Comment:
@sethherr Is this safe to delete? Doesn't seem to be in use anymore, but if we want to keep the empty partial around, I'll revert the commit.

Reply:
Yes. You can delete this. Good catch.